lib/pacemaker/pcmk_sched_probes.c

```diff
@@ -251,24 +251,24 @@ pcmk__probe_rsc_on_node(pe_resource_t *rsc, pe_node_t *node)
     // We've eliminated all cases where a probe is not needed, so now it is
     probe = probe_action(rsc, node);
 
-    /* Order the probe relative to the parent -- or the resource itself if
-     * cloned or a fence device when unfencing is used.
```

Comment:
guess this was wrong, s/cloned/not cloned/

Reply:
Yes, the comment was wrong